pkg/web/templates/transactions.html

```diff
@@ -22,7 +22,7 @@ <h1 class="font-bold text-2xl">Welcome!</h1>
   <section class="mb-12">
     <h1 class="font-bold text-2xl">Transaction Inbox</h1>
     {{ if or (.HasRole "Admin") (.HasRole "Compliance") }}
-    <div class="mt-6 flex justify-between items-center">
+    <div class="mt-6 flex flex-col sm:flex-row gap-y-3 sm:gap-y-0 sm:justify-between sm:items-center">
```

Comment:
The 2 buttons at the top of the transaction page were smooshed together on small screens. This fix will ensure that they appear stacked vertically on those small screens.